client/components/Auth/InputField.vue

```diff
@@ -1,10 +1,16 @@
 <template>
   <div>
-    <label class="font-semibold">{{ fieldName }}</label>
-    <ValidationProvider v-slot="{ errors, touched }" :vid="id" :rules="rules">
+    <label class="font-semibold" :for="fieldName">{{ fieldName }}</label>
```

Comment:
Seems like this file is using `==` where `===` should be. 

Is there a good reason for that? I don't think you intend to use `==`

Reply:
Seems like it.
As I didn't write `InputField`, I can't provide an answer as to why.